doc/data_structures.md

```diff
@@ -476,9 +476,13 @@ This API call is handled by LibSovToken build_payment_req_handler.
 
 [
     {
-        "address" : <str>,   // the payment address
-        "amount": <int>,    // the payment amount
+        "address" : <str>,   // the payment address correlating to the input utxo
```

Comment:
At the plugin level, there is no requirement that one of the input addresses be used for whatever amount returns as change from the transaction. Apps that use libsovtoken through the libindy payments API may elect to follow this practice, but it should not be required.

Reply:
@brentzundel could you please review?

Reply:
I read your code and believe I understood it. 
It is certainly the case that a remainder address is required. This remainder address can be any valid address, it does not need to be one of the input addresses from the transaction.